roles/logrotate/tasks/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+- block:
+    - name: Install logrotate
+      package:
+        name: logrotate
+        state: present
```

Comment:
`present` in this context means "not absent" instead of "current version". Regarding your actual point, few (if any) of our apt-get/package module uses have versions specified.